common/stacks/src/test/scala/weco/api/stacks/services/ItemLookupTest.scala

```diff
@@ -109,6 +109,79 @@ class ItemLookupTest
     }
   }
 
+  describe("bySourceIdentifiers") {
```

Comment:
Not for this patch, but it occurs to me that if we're going to have a MemoryItemLookup, we should really have generic tests to ensure it behaves the same way as ElasticItemLookup.

Reply:
Yes, I noticed that. Agree, will add in future PR.